.circleci/config.yml

```diff
@@ -43,8 +43,7 @@ jobs:
           command:
```

Comment:
Can we fix that?

<img width="521" alt="image" src="https://user-images.githubusercontent.com/5394573/182258423-211d54c4-7c95-43b6-ab88-585a6c62bb0e.png">